pom.xml

```diff
@@ -81,6 +84,92 @@
 
   <build>
     <plugins>
+      <plugin>
```

Comment:
Lines 87-107 are the actual change, everything else is just moving definitions around so they are in alphabetical order.